teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -183,6 +184,12 @@ interface InvestigationLocator {
     fun all(): Sequence<Investigation>
 }
 
+interface MuteLocator {
+    fun limitResults(count: Int): MuteLocator
+    fun forProject(projectId: ProjectId): MuteLocator
```

Comment:
I think forUser(id: UserId) would fit here nicely too

Reply:
From what I have tested, mutes do not have assignees, but I have added filter for the reported. Here is the result of the request with list of the supported locators:
```
Responding with error, status code: 400 (Bad Request).
Details: jetbrains.buildServer.server.rest.errors.LocatorProcessException: Error processing locator 'assignee:nn,count:100': Locator dimension [assignee] is unknown. Supported locator dimensions:
id - internal mute id (type: number)
test - test for which mute is assigned (type: test locator)
problem - problem for which mute is assigned (type: problem locator)
affectedProject - project affected by the mutes (type: project locator)
project - project in which mute is assigned (type: project locator)
creationDate - mute creation time (type: time condition)
unmuteDate - automatic unmute time (type: time condition)
reporter - muting user (type: user locator)
type - what is muted (type: one of test, problem)
resolution - unmute condition (type: one of manually, whenFixed, atTime)
```